src/eraftpb.rs

```diff
@@ -3,7 +3,7 @@
 
 // https://github.com/Manishearth/rust-clippy/issues/702
 #![allow(unknown_lints)]
-#![allow(clippy)]
+#![cfg_attr(feature = "cargo-clippy", allow(clippy::all))]
```

Comment:
Isn't `#![allow(unknown_lints)]` enough?

Reply:
No because protobuf generates code that clippy complains about, so you need to allow all lints to trigger on the generated protobufs.